src/docs/linea.mdx

```diff
@@ -159,12 +187,10 @@ Linea is a ZK-Rollup powered by Consensys zkEVM. The protocol team centrally ope
 
     Existing EVM-based tooling is supported such as ethers, web3.js, ethers-rs, hardhat, foundry and truffle.
 
-    [Pimlico](https://docs.pimlico.io/) provides smart account tools for ERC-4337.
-    
-    [Bicnomy](https://www.biconomy.io/) SDK provides solutions from onboarding to user engagement for a decentralized applications (dApps).
-    
-    [thirdweb](https://portal.thirdweb.com/wallet/smart-wallet) Smart Wallet SDK, implementing EIP-4337.
-    
     [Linea SDK](https://docs.linea.build/build-on-linea/linea-sdk) focuses on interacting with smart contracts on both Ethereum and Linea networks and provides custom functions to obtain message information.
 
+    [Linea Safe](https://safe.linea.build/welcome) is a multisig wallet for Linea.
```

Comment:
let's change it to:

[Linea Safe](https://safe.linea.build/welcome) is a white-label fork of Gnosis Safe deployed by the Linea team.